### DISCUSSION TITLE
Sort profiles by size

### DISCUSSION THREAD

**Discussion by Vorax1213** [belowaverage-org/SuperGrate#115] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
We have to right-click and select "Properties" on a profile to see its size and we cannot sort the profiles by size

**Describe the solution you'd like**
A new column with the sizes would be great !

**Additional context**
![image](https://github.com/belowaverage-org/SuperGrate/assets/82668876/94d5a30e-9ced-4a80-96a2-b325771a17e5)

**Comment by krisdb2009** [comment, marked as answer]
This is actually already implemented! Check it out:

![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/0589d5c3-69ab-4a7c-9ca8-2403e09bac4b)
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/4df9931e-cb24-4c64-9264-a77372123e4a)
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/fd237ed9-76fd-4ac1-9f4d-1b92b268bbdf)

**Reply by Vorax1213** [reply]
OH great! Sorry for the issue then, I should have seen it. Thanks !

**Reply by krisdb2009** [reply]
And if you want to save it so that column appears every time you launch Super Grate; after changing your column options, open "Settings" and click "Save to disk."

![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/8e74cdd1-20d6-4309-8210-d303e43222e1)
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/f50a0f37-398a-47f2-889f-a98fa068d383)

**Reply by krisdb2009** [reply]
> OH great! Sorry for the issue then, I should have seen it. Thanks !

No worries, helps to ask and get it documented!
